### PR TITLE
Creating a first global layout

### PR DESCRIPTION
**Pull request by EmileCalixte**
This pull request aims to create a first layout for the app, using Bootstrap and custom CSS rules.

### CODE REVIEW

File: components/view/navbar/Navbar.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+     *          [
+     *              'title'         => 'Item title',
+     *              'url'           => 'URL when item clicked',
+     *              'roleNeeded'    => 'role'   // Optionnal, if not set everyone will see it
+     *              'activeActions' => [        // Optionnal, if not set the item will never be shown as active
+     *                  'controller-name'   => [
+     *                      'action-name',
+     *                      'action-name',
+     *                      // OR
+     *                      '*'
```
Comment (alexisljn) [inline components/view/navbar/Navbar.php]: This should be an array OR a '*' but not both.

File: components/view/navbar/Navbar.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+     *                      'roleNeeded'    => 'role'   // Optionnal
+     *                  ]
+     *                  'divider' // A delimiter (horizontal grey line) in the dropdown
+     *                  [
+     *                      ...
+     *                  ]
+     *              ],
+     *              [
+     *                  ...
+     *              ]
```
Comment (alexisljn) [inline components/view/navbar/Navbar.php]: Indentation errors for others main items

File: views/layouts/main.php [hunk 3]
```diff
@@ -35,40 +36,22 @@
...
+                <form method="post" action="/site/logout" class="form-inline my-2 my-lg-0">
+                    <?= Html::hiddenInput(Yii::$app->request->csrfParam, Yii::$app->request->csrfToken) ?>
+                    <button class="btn btn-outline-danger my-2 my-sm-0" type="submit">Logout</button>
+                </form>
+            </div>
+        </nav>
+    </header>
 
     <div class="container">
         <?= Breadcrumbs::widget([
```
Comment (alexisljn) [inline views/layouts/main.php]: We don't use the Breadcrumbs widget

**Review by alexisljn — CHANGES_REQUESTED** [review]

**Review by GuenoleCHERUBIN — DISMISSED** [review]

File: components/view/navbar/Navbar.php (hunk 1 above)
Comment (EmileCalixte) [reply components/view/navbar/Navbar.php]: Well regarded !

File: components/view/navbar/Navbar.php (hunk 2 above)
Comment (EmileCalixte) [reply components/view/navbar/Navbar.php]: Yup, will fix it

File: views/layouts/main.php (hunk 3 above)
Comment (EmileCalixte) [reply views/layouts/main.php]: Yes, I'll remove it

**Review by alexisljn — APPROVED** [review]
Nothing to add

**Pull request merged**